Reply/app/src/main/res/layout/fragment_search.xml

```diff
@@ -0,0 +1,78 @@
+<!--
+  Copyright (c) 2020 Google Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+  in compliance with the License. You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software distributed under the License
+  is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+  or implied. See the License for the specific language governing permissions and limitations under
+  the License.
+  -->
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="?attr/colorSurface"
+        android:fitsSystemWindows="true"
```

Comment:
For consistency, could you remove this attr for
`app:paddingTopSystemWindowInsets="@{true}"`

There is also `paddingLeft|Right|BottomWindowInsets` data binding adapters that grab the window insets and apply them as padding to the view for you. Were using this approach to accomplish full screen as its easier to manage than trying to find the right combo of fitSystemWindows, etc.

Reply:
sure!